scripts/replace_template.sh

```diff
@@ -63,12 +63,12 @@ else
 fi
 echo "
       # Parameters to track
-      IntegrationNameField: !Ref \"AWS::StackName\"
```

Comment:
Why not `IntegrationName`?

Reply:
We define it as `stackName` there https://github.com/coralogix/integration-definitions/blob/master/integrations/cloudtrail-sns/v0.0.1/fields.yaml#L12

Reply:
we have it as **stackName** in definitions, but n the other hand we store it as IntegrationNameField in the database. Maybe the better option to make it all work together would be to update the definition, so it is IntegrationName in the definitions and also here. It might allow us to avoid some problems in the future if we try to stick to the same base schema for all the integrations, so they all have  IntegrationName

Reply:
I've [changed it](https://github.com/coralogix/eng-svc-extensions-api/blob/17348368b49fd355142404b932f78b5a0b416623/src/main/scala/com/coralogix/extensions/model/integration.scala#L2182) to store `stackName` in the db (it's backwards compatible).
Agree with updating the field in definitions to `IntegrationName`, will create a new PR.